VideoLocker/res/layout/registration_spinner_item.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="?android:attr/spinnerItemStyle"
+    tools:targetApi="17"
```

Comment:
Is this really needed ? I don't see an attribute that's targeted for API 17. Also, if we remove this line, we can get rid of the namespace inclusion as well i.e. `xmlns:tools="http://schemas.android.com/tools"`.